mercury.spec

```diff
@@ -47,6 +47,7 @@ Mercury devel
 %setup -q
 %patch1 -p1
 %patch2 -p1
+%patch3 -p1
```

Comment:
As alluded to on Skype, the alternative is to replace all of those lines with:

```
%autosetup
```

Then all future `Patch$n:` additions will automatically be patched without having to add a corresponding `%patch$n` here.